src/App.js

```diff
@@ -42,6 +42,11 @@ class App extends Component {
         domoticzLogin: '',
         domoticzPassword: ''
       },
+      configIdPagination: {
+        nbPages: 1,
```

Comment:
nbConfigIds maybe for consistency with the 2 props below ? or nbLayouts because of line 154